.web-docs/components/builder/vsphere-supervisor/README.md

```diff
@@ -185,6 +181,12 @@ items are listed below as well as the _optional_ configurations.
 - `bootstrap_data_file` (string) - Path to a file with bootstrap configuration data. Required if `bootstrap_provider` is not set to `CloudInit`.
   Defaults to a basic cloud config that sets up the user account from the SSH communicator config.
 
+- `guest_id` (string) - Name of the desired guest operating system identifier for source VM.
```

Comment:
Identifier of the desired guest operating system for the source VM

Reply:
`guest_os_type` is used in the `vsphere-iso` builder. Consider the same here and the same description.

Reply:
Changed to `guest_os_type`.